test/integration/vaultauthmethods_integration_test.go

```diff
@@ -159,7 +159,7 @@ func TestVaultAuthMethods(t *testing.T) {
 			},
 		},
 	}
-	// Apply all of the Auth Methods
+	// Apply all the Auth Methods
```

Comment:
I was wondering why my IDE kept flagging this line. Oh well. Learn something new every day.